pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
```

Comment:
What is `hatchling`, and how is it connected to `uv`? On first look it appears it's another different build system?

Reply:
uv as of today has no [build backend](https://packaging.python.org/en/latest/glossary/#term-Build-Backend): poetry has poetry-core, hatch has hatchling, pip has .. build? There is also a setuptools one. So essentially you have to pick one and some more. 

uv us currently developing one, but that looks still a bit rough. https://github.com/astral-sh/uv/issues/3957 Might already be enough though...

Reply:
Arch Linux packager here...

I fully support the switch to a proper PEP 517 build, and the `ruff` and `uv` tooling is almost universally recognized as better these days than black and pip and such. I don't have strong feelings about hatch/hatchling over poetry, flit, setuptools, or others, so that's fine I suppose.

On the other hand just a warning that the uv build backend is not only rough but controversial. I would avoid it for the foreseeable future as it introduces lots of binary requirements on the host environment when pure Python would do fine. It's very heavy weight for an application like this.

Reply:
That works for me, happy to use whatever works best for folks building the actual packages :-)